### PR TITLE
Reverting the revert that removed partial credit info

### PR DESCRIPTION
**Pull request by srpearce**
Don't review this yet - still testing

### CODE REVIEW

**Comment by srpearce** [comment]
@catong @mhoeber I think I have this right. I've reverted so that all the partial credit info is there for checkbox, custom Python-evaluated input, multiple choice, and numeric input problems, but not dropdown. Feel free to verify that it looks right. It built fine (including images) when I ran `make html`.

Assuming I should hold off merging this until next week's release (scheduled for Wednesday instead of Tuesday - I set the date in the change log to September 30).

File: en_us/shared/exercises_tools/checkbox.rst [hunk 1]
```diff
@@ -371,6 +373,177 @@ or the advanced editor. For an overview of hints in problems, see
...
+
+*********************************************
+Awarding Partial Credit in a Checkbox Problem
+*********************************************
+
+You can configure a checkbox problem to award partial credit to learners
+who submit an answer that is partly correct. You must use the `Advanced Editor
+<Use the Advanced Editor to Edit a Checkbox Problem>`_ to configure partial
+credit.
+
```
Comment (catong) [inline en_us/shared/exercises_tools/checkbox.rst:386]: @srpearce In the original I think there was a "Partners only" condition note that read: 'Support for partial credit problems in courses on edx.org and edX Edge is provisional. Ensure that you test such problems thoroughly before releasing them to learners. For more information, contact your edX Partner Manager."

Comment (catong) [inline en_us/shared/exercises_tools/checkbox.rst:386]: This was in @mhoeber 's 2nd partial credit PR: https://github.com/edx/edx-documentation/pull/525, a follow-up to his original https://github.com/edx/edx-documentation/pull/513

Comment (srpearce) [inline en_us/shared/exercises_tools/checkbox.rst:386]: Weird - that text doesn't appear in the text you removed in your revert [PR 533](https://github.com/edx/edx-documentation/pull/533). Since I'm just undoing the changes that you made, where would that text have gone? :-\ I can add it back in, but now I'm concerned that there might be other similar problems.

Comment (catong) [inline en_us/shared/exercises_tools/checkbox.rst:386]: I approached the PRs in reverse order; I was able to revert #525, so the revert of those changes became "master" and aren't visible in the version against which I had to make my manual changes to revert the changes Mark made in #533. Mark's 2nd PR (#525) should show you any additional changes.
